alerts/cluster-monitoring-operator/ClusterMonitoringOperatorDeprecatedConfig.md

```diff
@@ -0,0 +1,33 @@
+# ClusterMonitoringOperatorDeprecatedConfig
+
+## Meaning
+
+This alert fires when a deprecated config is used in the config map `openshift-monitoring/cluster-monitoring-config`.
+
+## Impact
+
+Using a deprecated config is not recommended because the config has no effect and
+it may cause `cluster-monitoring-operator` go in degraded state when the config
+is removed in a future release.
+
+## Diagnosis
+
+- Check the __Description__ and __Summary__ sections of the alert to identify
+the
+  deprecated config as in the following example:
+
+  __Description__
+
+  `The configuration field k8sPrometheusAdapter in
+  openshift-monitoring/cluster-monitoring-config was deprecated in 4.16 and has
+  no effect, you can remove it.`
```

Comment:
I think we should remove `you can remove it` from the alert description.
Now we have a runbook + in some cases we also need to migrate the config...

Reply:
Ideally the user shouldn't even have to read the runbook so I'd update the alert description to say the same about migrating the config.